From: Rainer Gerhards <rgerhards@adiscon.com>
Date: Tue, 19 Aug 2014 16:04:52 +0200
Subject: fix build problem on platforms without atomic instructions

Thanks to Michael Biebl for his very helpful analysis of the bug.
---
 runtime/atomic.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/atomic.h b/runtime/atomic.h
index 49c17d8..d59858a 100644
--- a/runtime/atomic.h
+++ b/runtime/atomic.h
@@ -221,7 +221,7 @@
 		return(val);
 	}
 
-#	define DEF_ATOMIC_HELPER_MUT64(x)  pthread_mutex_t x
+#	define DEF_ATOMIC_HELPER_MUT64(x)  pthread_mutex_t x;
 #	define INIT_ATOMIC_HELPER_MUT64(x) pthread_mutex_init(&(x), NULL)
 #	define DESTROY_ATOMIC_HELPER_MUT64(x) pthread_mutex_destroy(&(x))
 #endif /* #ifdef HAVE_ATOMIC_BUILTINS64 */
